web/multinode/src/app/store/nodes.ts

```diff
@@ -106,9 +106,13 @@ export class NodesModule implements Module<NodesState, RootState> {
      */
     public async fetchOnline(ctx: ActionContext<NodesState, RootState>): Promise<void> {
         const nodes = ctx.state.selectedSatellite ? await this.nodes.listBySatellite(ctx.state.selectedSatellite.id) : await this.nodes.list();
-        
-        const onlineNodes = nodes.filter((node: Node) => node.status === NodeStatus['online']);
-        
+        let onlineNodes;
+        if (nodes || Array.isArray(nodes)) {
```

Comment:
It seems that you want to ensure that nodes are an Array to go into this conditional because you're calling an Array method then `if (Array.isArray(nodes))` is enough.

Reply:
Actually one test was failing for if the nodes array become `undefined` So I wanted to make sure. Removed the nodes check.